Extensions/Physics2Behavior/physics2runtimebehavior.ts

```diff
@@ -741,11 +754,27 @@ namespace gdjs {
       this._objectOldX = this.owner.getX();
       this._objectOldY = this.owner.getY();
       this._objectOldAngle = this.owner.getAngle();
+
+      gdjs.physics2.computeCurrentContactsFromStartedAndEndedContacts(
+        this.currentContacts,
+        this.contactsStartedBetweenFrames,
+        this.contactsEndedBetweenFrames
+      );
     }
 
     doStepPostEvents(runtimeScene) {
       this._updateBodyFromObject();
 
+      // Reset contacts that happened in-between frames
+      this.contactsStartedBetweenFrames.splice(
+        0,
+        this.contactsStartedBetweenFrames.length
+      );
```

Comment:
Setting the length to `0` is easier to read (maybe faster but I'm not sure)

Reply:
Yep, use `length = 0`, this is handled well by JS engines.

Reply:
The arrays can be cleared at the beginning of `doStepPreEvents`. It will allow to write tests more easily. It won't change how it behaves as these arrays are not used during the rending.